src/components/registerForm.vue

```diff
@@ -0,0 +1,124 @@
+<template>
+    <div class="mx-auto p-5 rounded-3xl w-min" style="background-color: #313030;">
+        <div class="p-5">
+            <h1>Register</h1>
+            <p>Fill all fields</p>
+            <div class="mt-12">
+                <form>
+                    <p v-if="parsedMessage" class="text-red-600 w-full mb-4 text-xs">{{ parsedMessage }}</p>
```

Comment:
Should be placed on the submit button instead

Reply:
Changes added